### PR TITLE
chore: update pr template

### PR DESCRIPTION
**Pull request by jrschumacher**
Update PR template to recommend adding label

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-changes: _rev_
+_Please ensure to label (via GitHub labels) the PR with either patch, minor, or major._
 
 ### Proposed Changes
-_Please use the Jira Key or NOREF followd by the changes_
+_Please use the Jira Key followd by the changes_
```
Comment (avkvirtru) [inline .github/pull_request_template.md:4]: typo: "followed" is missing an "e"

**Review by avkvirtru — APPROVED** [review]
LG2M 👍 except the minor typo

File: .github/pull_request_template.md (hunk 1 above)
Comment (jrschumacher) [reply .github/pull_request_template.md:4]:
```suggestion
_Please use the Jira Key followed by the changes_
```

**Comment by virtru-github-public** [comment]
:rocket: PR was released in [`v1.16.0`](https://github.com/virtru/virtuoso-design-system/releases/tag/v1.16.0) :rocket:

**Pull request merged**